tests/longevity-large-partition-1TB-7days.yaml

```diff
@@ -0,0 +1,77 @@
+test_duration: 10080
```

Comment:
Please change the name to be 4days.
Also please change the duration to be 4 days.